runtime/autoload/man.vim

```diff
@@ -71,16 +65,53 @@ function! man#read_page(ref) abort
   call s:read_page(path)
 endfunction
 
+" Handler for s:system() function.
+function! s:system_handler(jobid, data, event) abort
+  if a:event == 'stdout'
+    let self.output .= join(a:data, "\n")
+  elseif a:event == 'exit'
```

Comment:
Could this be simplified to just an `else` because it can only be called on only `stdout` or `exit`?